helm-charts/sep-service/values.yaml

```diff
@@ -0,0 +1,134 @@
+fullName: anchor-platform
+namespace: sandbox
```

Comment:
I wouldn't provide default here. Many charts use `namespaceOverride` and default to empty one. That may be the safest way to do it. WDYT?